src/components/AccordionSection/AccordionSection.test.tsx

```diff
@@ -49,11 +51,13 @@ describe("AccordionSection ", () => {
       />
     );
     wrapper.find(".p-accordion__tab").at(0).simulate("click");
-    expect(onTitleClick.mock.calls[0][0]).toBe(true, MOCK_UUID);
```

Comment:
Typescript insists that `toBe` only takes a single argument, so I'm not sure how these tests passed.